scaffolds/streamlit_on_cloudrun/llm_chatbot/build_app.ipynb

```diff
@@ -0,0 +1,621 @@
+{
```

Comment:
<p>I think while the readability is better when the explanations of these blocks is in Markdown. I think most people will find themselves running these cells multiple times and this will mess up the order of the functions in the app.py file. My recommendation is to create a single cell with %writefile app.py with all the code that goes in it and have the functions illustrated separately outside.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/524/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>